manual/src/tutorials/effectexamples.etex

```diff
@@ -0,0 +1,666 @@
+\chapter{Effect Handlers} \label{c:effects-tutorial}
+%HEVEA\cutname{effects-tutorial.html}
+
+Effect handlers are a mechanism for modular programming with user-defined
+effects. Effect handlers allow the programmers to describe
+\textit{computations} that \textit{perform} effectful \textit{operations}, whose
+meaning is described by \textit{handlers} that enclose the computations. Effect
+handlers are a generalization of exception handlers, and enable non-local
+control-flow mechanisms such as resumable exceptions, lightweight threads,
+coroutines, generators and asynchronous I/O to be composably expressed. In this
+tutorial, we shall see how some of these mechanisms can be built using effect
+handlers.
+
+\section{s:effects-basics}{Basics}
+
+To understand the basics, let us define an effect that exchanges an integer,
```

Comment:
"An effect that exchanges an integer" doesn't sound very clear to me. I would suggest "an effect (that is, an operation) that takes an integer argument and returns an integer result". I would also add "We name this effect `Xchg`."